package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components-wdio-test",
-  "version": "1.5.0",
+  "version": "1.7.0",
```

Comment:
Do we really need to update twice the version number (considering PR #28)?
I would suggest to move again to version 1.6.0, so both changes are included on a single new NPM version